src/sentry/utils/eventuser.py

```diff
@@ -125,6 +125,21 @@ def from_snuba(result: Mapping[str, Any]) -> EventUser:
             user_ident=result.get("user_id"),
         )
 
+    @classmethod
+    def for_tags(cls, project_id: int, values):
+        """
+        Finds matching EventUser objects from a list of tag values.
+
+        Return a dictionary of {tag_value: event_user}.
+        """
+        projects = Project.objects.filter(id=project_id)
+        result = {}
+        for value in values:
+            keyword_filters = {value.split(":", 1)[0]: value.split(":", 1)[-1]}
+            eventusers = EventUser.for_projects(projects, keyword_filters)
```

Comment:
Nit: Tbh we should probably change `for_projects` to just return an `EventUser` dataclass or None, rather than having to unwrap it like this.